src/style.css

```
@@ -1,3 +1,65 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo {
+  text-decoration: none;
+  height: 40px;
+}
+
+.nav__item {
+  margin-left: 20px;
+}
+
+.nav__link {
+  align-items: center;
+  text-decoration: none;
+  display: flex;
+  height: 60px;
+  position: relative;
+  text-transform: uppercase;
+  font-size: 12px;
+  color: black;
+}
+
+.is-active {
+  color: #00acdc !important;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.nav__link:active {
```

You can combine selectors with the same styles 
```suggestion
.nav__link:hover,
.nav__link:active {
```